Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -69,5 +74,16 @@ def commit(self):
             distances = inv(distances)
         self.send("Distances", distances)
 
+    def send_report(self):
+        norm, normopt = self.normalization_method, self.normalization_options
+        inv, invopt = self.inversion_method, self.inversion_options
+        parts = []
+        if inv:
+            parts.append('inversion ({})'.format(invopt[inv][0]))
+        if norm:
+            parts.append('normalization ({})'.format(normopt[norm][0]))
+        self.report_items('Model parameters',
+                  (('Transformation', ', '.join(parts).capitalize() or 'None'),))
+
     def _invalidate(self):
-        self.commit()
+        self.commit()
```

Comment:
No need to (read: _don't_) delete the final newline character in the file. Git and GitHub both warn against it. :smile: